HashRegistrarSimplified.sol

```diff
@@ -189,28 +232,44 @@ contract Registrar {
         AuctionStarted(_hash, newAuction.registrationDate);      
     }
 
-    // Allows you to open multiple for better anonimity
+    /**
+     * @dev Start multiple auctions for better anonymity
+     * @param _hashes An array of hashes, at least one of which you presumably want to bid on
+     */
     function startAuctions(bytes32[] _hashes)  {
         for (uint i = 0; i < _hashes.length; i ++ ) {
             startAuction(_hashes[i]);
         }
     }
     
+    /**
+     * @dev Hash the values required for a secret bid
+     * @param hash The node corresponding to the desired namehash
+     * @param owner The address which will own the 
+     * @param value The bid amount
+     * @param salt A random value to ensure secrecy of the bid
+     * @return The hash of the bid values
+     */
     function shaBid(bytes32 hash, address owner, uint value, bytes32 salt) constant returns (bytes32 sealedBid) {
         return sha3(hash, owner, value, salt);
     }
     
     /*
     ## Blind auction for the desired hash
-
+    
     Bids are sent by sending a message to the main contract with a hash and an amount. The hash 
     contains information about the bid, including the bidded hash, the bid amount, and a random 
     salt. Bids are not tied to any one auction until they are revealed. The value of the bid 
     itself can be masqueraded by changing the required period or sending more than what you are 
     bidding for. This is followed by a 24h reveal period. Bids revealed after this period will 
     be burned and the ether unrecoverable. Since this is an auction, it is expected that most 
-    public hashes, like known domains and common dictionary words, will have multiple bidders pushing the price up. 
+    public hashes, like known domains and common dictionary words, will have multiple bidders 
+    pushing the price up. 
     */ 
+    /**
```

Comment:
Like this? 
```/**
     * ## Blind auction for the desired hash
     * 
     * Bids are sent by sending a message to the main contract with a hash and an amount. The hash 
     * contains information about the bid, including the bidded hash, the bid amount, and a random 
     * salt. Bids are not tied to any one auction until they are revealed. The value of the bid 
     * itself can be masqueraded by changing the required period or sending more than what you are 
     * bidding for. This is followed by a 24h reveal period. Bids revealed after this period will 
     * be burned and the ether unrecoverable. Since this is an auction, it is expected that most 
     * public hashes, like known domains and common dictionary words, will have multiple bidders 
     * pushing the price up. 
     *  
     * @dev Submit a new sealed bid
     * @param sealedBid A sealedBid, created by the shaBid function
     */
```